src/components/TeamMemberModal.astro

```diff
@@ -0,0 +1,138 @@
+---
+const { teammate } = Astro.props
+---
+
+<div 
+    class="team__member-modal" 
+    role="dialog" 
+    aria-labelledby={`dialog${teammate.id}title`}
+    aria-describedby={`dialog${teammate.id}description`}
```

Comment:
@Oluwa-nifemi Shouldn't these have spaces between them if you're trying to form a sentence